core/src/com/unciv/models/ruleset/unique/Countables.kt

```diff
@@ -33,6 +33,9 @@ object Countables {
             return civInfo.cities.sumOf { it.cityConstructions.getBuiltBuildings()
                 .count { it.matchesFilter(placeholderParameters[0]) } }
 
+        if (countable.equalsPlaceholderText("[] Tiles"))
+            return gameInfo.tileMap.tileList.count { it.matchesFilter(placeholderParameters[0]) }
```

Comment:
Tiles in the *entire map*?! That's both very unintuitive, and I can't really see a use case for this
All the other filters are for civ-wide counts, not map-wide

Reply:
So it should be only civ-wide?

What should we use instead for obtaining map area, height, width or amount of land and water tiles?

Reply:
Who even wants that?

Reply:
Please explain me your advices in more straightforward and unambiguous way.

So I understand that it tile-counting should be civ-wide, not map-wide. Am I right?